dbc2val/dbcfeederlib/dbc2vssmapper.py

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python3
 
 ########################################################################
-# Copyright (c) 2020 Robert Bosch GmbH
+# Copyright (c) 2020,2023 Robert Bosch GmbH
```

Comment:
@SebastianSchildt - do we somewhere have any guidelines on how to handle "years" in copyright? As of today we have for dbcfeeder two types of license statements, one (like here) stating Bosch copyright, others have a more generic "Contributors to the Eclipse Foundation". Then for year, when shall we add a new year like here. Always?

Reply:
The same section also states for the newer copyright header's 1st line:

> The year is expressed as a single year or a range spanning the year of the very initial assertion and that of the most recent change.

:-)